docs/content/guides/rows/row-sorting.md

```diff
@@ -1,176 +1,609 @@
 ---
 id: 6o0zftmc
-title: Row sorting
-metaTitle: Row sorting - JavaScript Data Grid | Handsontable
-description: Sort your data in ascending, descending, or a custom order, across one or multiple rows of the data grid.
+title: Rows sorting
+metaTitle: Rows sorting - JavaScript Data Grid | Handsontable
+description: Sort rows alphabetically or numerically, in ascending, descending or a custom order, across one or multiple columns.
 permalink: /row-sorting
 canonicalUrl: /row-sorting
+tags:
+  - row sorting
+  - column sorting
+  - columnSorting
+  - multicolumn sorting
+  - multi-column sorting
+  - multiColumnSorting
+  - row ordering
+  - order by
+  - sort ascending
+  - sort descending
+  - highest to lowest
+  - unordered data
+  - sortEmptyCells
+  - data sorting
+  - sort data
+  - sort rows
+  - sort columns
+  - sorting
 react:
   id: h4jfevxj
-  metaTitle: Row sorting - React Data Grid | Handsontable
+  metaTitle: Rows sorting - React Data Grid | Handsontable
 searchCategory: Guides
 ---
 
-# Row sorting
+# Rows sorting
 
-Sort your data in ascending, descending, or a custom order, across one or multiple rows of the grid.
+Sort data alphabetically or numerically, in ascending, descending or a custom order, across one or multiple columns.
 
 [[toc]]
 
 ## Overview
 
-The [`ColumnSorting`](@/api/columnSorting.md) plugin works as a middleware between the data source and the Handsontable rendering module.
+With sorting, you can easily rearrange rows of data, based on the values in specified columns.
+This is particularly useful for analyzing and organizing large datasets, which helps you identify patterns and trends.
 
-The plugin can:
-* Map indices of displayed rows `visual indices` to the indices of corresponding rows in the datasource `physical indices` and vice versa
-* Alter the order of rows presented to a user without changing the datasource’s internal structure
+You can sort data in different ways:
+- Alphabetically, numerically, or based on a [custom data type](@/guides/cell-types/cell-type.md).
+- In ascending, descending, or a [custom order](#add-a-custom-comparator).
+- By a single column, or by [multiple columns](#sort-by-multiple-columns).
+- Using Handsontable's [UI](#demo) or [API](#control-sorting-programmatically).
 
-The sort operation is performed using a [stable sort algorithm](https://en.wikipedia.org/wiki/Sorting_algorithm#Stability) regardless of the browser you use or the size of the data set which you sort.
+Data is sorted just visually. Your source data remains in the original order.
 
-::: tip
-Don't enable the [`ColumnSorting`](@/api/columnSorting.md) and [`MultiColumnSorting`](@/api/multiColumnSorting.md) plugins simultaneously.
-:::
+### Demo
 
-## Basic plugin configuration
+Click on any column name. Based on the values in that column,
```

Comment:
> Click on a column name in the column header 

(I think you meant "a column name in **a** column header) – "column name" is self-explanatory, no need to repeat that.

> to sort the values in the column

That's not true: you don't sort "the values in the column", you sort all the values in all the columns. It's the same problem as with calling the feature "column sorting" instead of "row sorting".

> **,** or go back to the original order.

That comma implies that you either "click" or "go back".

I'd go for:
> Click on a colum name to sort the values in ascending (↑) or descending (↓) order, or to go back to the original order.